apps/web/src/pages/Teams/TeamMembersPage/components/types.ts

```diff
@@ -7,3 +7,49 @@ export type TeamMemberFormValues = {
   userDisplay: Scalars["UUID"]["output"];
   roles: Array<Scalars["UUID"]["output"]>;
 };
+
+export const TeamMembersPage_TeamFragment = graphql(/* GraphQL */ `
```

Comment:
Would it be possible to move the actual query definition to a different file. Or rename this? I would only expect types to show up in this file.

Reply:
Sure, done as a part of https://github.com/GCTC-NTGC/gc-digital-talent/pull/9765/commits/e2d2ca63ce7e71914d83d5374ba115578c5e1ee4